### ISSUE TITLE
Create User Login / Registration

### ISSUE THREAD

**Issue by NitinNair89** [NitinNair89/TheHungryRecipesApp#15]
**Is your feature request related to a problem? Please describe.**
FEATURE REQUEST.
There's no provision for visitors to mark any recipes as favorite.

**Describe the solution you'd like**
A user login / registration page asking for minimum details from user - Email Address and Password

**Describe alternatives you've considered**
MONGODB for storage. Any other alternatives? You can suggest.

**Comment by NirmalKumarJoshi** [comment]
I would like to work on it under Hacktoberfest. Please assign this to me. :)

**Comment by macbookfilip** [comment]
Hello, I would like to try to working on this issue. Could you assign it to me please? Thank you

**Comment by NitinNair89** [comment]
Sorry @macbookfilip . @NirmalKumarJoshi is already working on this one.

Feel free to open bugs/issues and share ideas for this project. :slightly_smiling_face:

**Comment by NitinNair89** [comment]
> hi @NitinNair89 can I work on this issue for hacktoberfest?

Hi @koshimagoyal Sure you can.
Can you provide an estimate (in days) on delivery? As you can see, others wanted to work on the issue but never delivered. So just wanted to make sure. 🙂

**Comment by NitinNair89** [comment]
@koshimagoyal I want you to prepare a wireframe/mockup first before starting development. Once the UI is approved and final, you can proceed with development.
MongoDB integration you can do it later (we will create separate task for it). You can also head over to the Projects tab and check/create tasks for the current version of this project.
Does it sound good?

**Comment by NitinNair89** [comment]
@koshimagoyal Do you have any designs in your mind which you can create? I am open to suggestions, but let's not use others designs without approvals in place. 🙂 

The main theme of the app is green-ish, so you can use that. Maybe the app name/logo on the left and the login/registration box on the right as seen in the Dribble link would look cool.

Let's just use app name as the text without any images. Then in the future iterations, you can work on improving the page. Thoughts?

**Comment by koshimagoyal** [comment]
@NitinNair89 have created page but you will have to remove navbar from app.component.html so that login/registration page does not show navbar. Either you can use it in all files or you can go for one single file and their you can define routing. Also if you are okay with navbar on login/register page then I can create pull request.

**Comment by NitinNair89** [comment]
> @NitinNair89 have created page but you will have to remove navbar from app.component.html so that login/registration page does not show navbar. Either you can use it in all files or you can go for one single file and their you can define routing. Also if you are okay with navbar on login/register page then I can create pull request.

Yea agreed. I think navbar should have its own component file and login/registration page UI should just have the app name with the appropriate forms. 

Please proceed 👍🏻
